engineering/frontend/logging-monitoring.md

```diff
@@ -0,0 +1,20 @@
+# Logging and Monitoring
+
+This document in intended to let you know how we monitoring the frontend applications at Dwarves
+
+## Tracing errors
+
+We choose [Sentry](https://sentry.io/) to catch the error on both Frontend and Backend so our dev team can monitor whole system in one.
+
+- [x] Integrating to staging/dev server first so we can catch an issue before it become a production bug
+- [x] Sending email, and notification to responsibility developers ASAP when production issue appear to take quick action
+- [x] Linking errors to management tools to don't miss these errors
+- [x] Make sure status on Sentry issues are up to date
+
+
+## Tracking events and monitoring performance
+
+We install [Google Analytic](https://analytics.google.com/) to our projects as default to track all user interaction and monitoring page performance . Beside that, we will add custom events and set goals to our Google Analytic base on Marketing team requirements
```

Comment:
@thminhVN Should we add some tools like [PageSpeed Insights](https://pagespeed.web.dev/) or Lighthouse and how to use them to monitor performance? Or in addition, it's also monitored by recording performance in devtool and analyzed based on frames, scripting, rendering...

Reply:
Agree with An even though we normally do the ad-hoc test rather than including the check in CI pipeline.